roles/ec2-ami-base/README.md

```diff
@@ -0,0 +1,103 @@
+opg-ansible-ami-base
```

Comment:
I have looked into the opg-ansible-roles/Readme file and the different Readmes in the separate roles, and I don't actually see anything in those files that sets this README apart from those.
please expand